pwa/src/templates/templateParts/header/HeaderTemplate.tsx

```diff
@@ -154,13 +154,21 @@ export const HeaderTemplate: React.FC<HeaderTemplateProps> = ({ layoutClassName
       <div className={styles.headerMiddleBar}>
         <Container layoutClassName={styles.primaryNavContainer}>
           <div className={clsx(styles.logoContainer, styles.logoDesktop)}>
-            <div onClick={() => navigate("/")} className={styles.organizationLogo} />
+            <Image
+              alt="Logo van de Gemeente Rotterdam"
+              src="images/LogoRotterdam.svg"
+              className="utrecht-img--scale-down"
+            />
           </div>
 
           <PrimaryTopNav
             mobileLogo={
               <div className={clsx(styles.logoContainer, styles.logoMobile)}>
-                <div onClick={() => navigate("/")} className={styles.organizationLogo} />
+                <Image
+                  alt="Logo van de Gemeente Rotterdam"
+                  src="images/LogoRotterdam.svg"
+                  className="utrecht-img--scale-down"
+                />
```

Comment:
By making this src attribute hardcoded (instead of setting it via the design token and css) the logo will _always_ be that of Gemeente Rotterdam.

Because of that, when changing to a new theme (e.g. Utrecht), the Rotterdam logo still shows, which is not intended.